searx/static/themes/oscar/js/searx.min.js

```diff
@@ -1,4 +1,4 @@
-/*! oscar/searx.min.js | 01-06-2021 | https://github.com/searxng/searxng  */
+/*! oscar/searx.min.js | 02-06-2021 | https://github.com/searxng/searxng  */
```

Comment:
bikeshedding: this patch can be moved to the commit which includes the build products [static].

Reply:
Done.